stablehlo/dialect/Version.h

```diff
@@ -38,7 +38,7 @@ class Version {
   static FailureOr<Version> fromString(llvm::StringRef versionRef);
 
   /// Return a Version representing the current VHLO dialect version.
-  static Version getCurrentVersion() { return Version(0, 14, 5); }
+  static Version getCurrentVersion() { return Version(0, 14, 6); }
```

Comment:
Opening comment to tag 0.14.5, can resolve and submit when completed.

Reply:
Tagged.